python/tox.ini

```diff
@@ -24,6 +24,7 @@ deps =
     coverage
     mock
     pytest
+    typing-extensions
```

Comment:
not very familiar with tox but is there a way to specify this is required for python 3.7?

Reply:
I actually think python 3.7 may come with typing-extensions by default. Let me confirm that and we can just remove this dep here entirely.

Reply:
Confirmed and updated!